src/model/block/ISelectOneResponseBlockConfig.ts

```diff
@@ -20,9 +20,35 @@
 import {IBlockConfig} from '../..'
 
 export interface ISelectOneResponseBlockConfig extends IBlockConfig {
-  prompt: string
-  question_prompt?: string
-  choices: StringMapType
+  prompt?: Uuid
+  question_prompt?: Uuid
+  choices: Record<string, Choice>
```

Comment:
I think I've missed this from RC3 https://floip.gitbook.io/flow-specification/v/drafts-1.0.0-rc3-1/changelog
so `config.choices` becomes array of choices now

- [ ] update choices based on RC3

Reply:
@bulatgab , I've addressed that in this PR https://github.com/FLOIP/flow-runner/pull/85

Reply:
in fact, I need to have the schema working for my ticket

Reply:
- [ ] merge my PR if you agree
- [ ] publish this to npm (I changed the version in my PR to `1.0.0-rc4.10`)